ontology/doid.md

```diff
@@ -43,6 +43,13 @@ license:
   url: https://creativecommons.org/publicdomain/zero/1.0/
   label: CC0 1.0 Universal
 usages:
+  - user: https://disease-ontology.org/community/collaborators
+    description: View the DO's ~ 300 contributors, collaborators and use cases
+    examples: 
+      - url: https://www.alliancegenome.org
```

Comment:
Basically to "prove" usage of DO, we require example URLs to it would be best if you could use an example like `https://www.alliancegenome.org/disease/DOID:14323` and use the description to describe what the user sees on that page.